### ISSUE TITLE
Specifying snippet

### ISSUE THREAD

**Issue by lorensr** [partyparrot/codetours#1]
> 1. Commit hash and line numbers - easiest to implement, most brittle, relies on PRs to update
> 2. Commit hash and search terms for start/end - need to update commit manually, and you can check if it is still relevant
> 3. Search terms and branch name - updates automatically, might break if search terms are not found
> 4. Search terms and branch name - we follow every commit and notify when relevant lines are modified

I prefer branch. Can you do line numbers and keep track of how those line numbers change based on git? So I specify lines 10-20 on master/file.js, and when there's a new commit that adds two lines at line 15, my YAML/JSON gets auto-changed to lines 10-22? If not, I'd prefer search terms, and notifying only when the search term breaks.

**Comment by stubailo** [comment]
Yeah I think we'll only be able to do the simplest approach -- 1 -- for the hackathon, since we ended up focusing on the reader experience and didn't want to work on building out CI-type functionality. But I like the idea of specifying line numbers and trying to track how they change.
